conformance/utils/suite/suite.go

```diff
@@ -219,6 +225,12 @@ func (test *ConformanceTest) Run(t *testing.T, suite *ConformanceTestSuite) {
 		}
 	}
 
+	// check that the test should not be skipped
+	if sets.New(suite.SkipTests...).Has(test.ShortName) {
```

Comment:
Completely understand why you did this because it's exactly what I linked to, but actually intended something slightly different. Initializing this in every test run is equivalent to looping through the list each time. Instead I'd recommend using `sets.String` as the type of the `SkipTests` field so this line can become `if suite.SkipTests.Has(test.ShortName)`. (May also need a nil check).

Reply:
Yeah, I understand. I've left the slice of strings in the options and used the sets in the `ConformanceTestSuite`, so that the set is initialized only once (at `ConformanceTestSuite` initialization time), but the user can specify the set of tests to skip as a slice of strings through the options.

Reply:
There is no need to check nil. Even passing a nil slice doesn't cause panic.